site/public/js/forum.js

```diff
@@ -2417,6 +2417,37 @@ function clearReplyBoxAutosave(replyBox) {
     }
 }
 
+function setupDisableReplyThreadForm() {
+    const threadPostForms = document.querySelectorAll('.thread-post-form');
+
+    threadPostForms.forEach((form) => {
+        //For all thread forms either reply's or posts, ensure that when text area is empty, the submit button appears to be disabled
+        const textArea = form.querySelector('textarea');
+        const submitButton = form.querySelector('input[type="submit"]');
+
```

Comment:
I had a thought:
```suggestion
pbid = $('#edit-user-post').find('.thread-post-form').data('post_box_id');
```
will get the post_box_id for the edit post form (currently 0). Then below if you look for reply boxes pbid and pbid+1, that should resolve the conflicts with my PR and this should be ready to merge.